doc/ip/revoker.md

```diff
@@ -0,0 +1,15 @@
+# CHERIoT hardware revocation engine
+
+This block is integrated from the [CHERIoT Safe repository](https://github.com/microsoft/cheriot-safe/blob/main/src/msft_cheri_subsystem/msftDvIp_mmreg.sv).
+It essentially sweeps memory for revoked capabilities.
```

Comment:
It is the thing that does the revocation, it is looking for capabilities to freed memory.

Reply:
Thanks, I've reworded this.